scripts/launch_validator_node.sh

```diff
@@ -91,6 +91,9 @@ function launch_nodes {
   
   # add-validator node
   become_a_validator
+
```

Comment:
@karthikeyan-cere 
Double-check the ticket description. Container removing can be added to the run command.

Reply:
Updated command

Reply:
Remove extra line here

Reply:
@karthikeyan-cere 
Remove extra line.

Reply:
Done.